hphp/runtime/vm/jit/vasm-simplify-arm.cpp

```diff
@@ -84,17 +84,38 @@ bool simplify(Env& env, const cmovq& inst, Vlabel b, size_t i) {
 
 ///////////////////////////////////////////////////////////////////////////////
 
-bool simplify(Env& env, const loadb& inst, Vlabel b, size_t i) {
-  return if_inst<Vinstr::movzbl>(env, b, i + 1, [&] (const movzbl& mov) {
-    // loadb{s, tmp}; movzbl{tmp, d}; -> loadzbl{s, d};
-    if (!(env.use_counts[inst.d] == 1 &&
-          inst.d == mov.s)) return false;
+template<Vinstr::Opcode op>
+bool is_next(const Env& env, Vlabel b, size_t i)
```

Comment:
Can you rename this to is_inst?
This function is testing the same instruction i that it takes, not the next one, so is_next is misleading.